stack/global-hints.yaml

```diff
@@ -1111,3 +1111,40 @@ ghc-7.10.1:
   pretty: 1.1.2.0
   template-haskell: 2.10.0.0
   directory: 1.2.2.0
+ghc-9.4.0.20220523:
```

Comment:
Are you missing `Win32`?

Reply:
Seems like it! I couldn't use `update-global-hints.hs` for the alpha, will check if i can find the Win32 version somewhere.

Reply:
Found it here https://downloads.haskell.org/ghc/9.4.1-rc1/docs/libraries/Win32-2.12.0.0/Graphics-Win32.html